x/xos/env.go

```diff
@@ -0,0 +1,12 @@
+package xos
+
+import "os"
+
+// GetenvDefault retrieves the value of the environment variable named by the key.
+// It returns the value, which will be set to fallback if the variable is empty.
+func GetenvDefault(key, fallback string) string {
```

Comment:
This function is never used.

Reply:
And I would recommend to use Viper for the default value management

Reply:
I left it if someone need this function (eg for testing)